src/patterns/FiltersInline/FilterButton.tsx

```diff
@@ -0,0 +1,215 @@
+import * as Popover from '@radix-ui/react-popover';
+import classnames from 'classnames';
+import React from 'react';
+import Button from '../../components/Button/Button';
+import { ButtonVariants } from '../../components/Button/types';
+import Drawer from '../../components/Drawer/Drawer';
+import Filter from '../../components/Filter/Filter';
+import FilterHeader from '../../components/Filter/FilterHeader';
+import FilterInput from '../../components/Filter/FilterInput';
+import { SSRMediaQuery } from '../../providers/SeldonProvider/utils';
+import { px } from '../../utils';
+import FilterMenu from '../FilterMenu/FilterMenu';
+import { FilterButtonDisplay } from './FilterButtonDisplay';
+import { FilterDropdownMenuDesktop, FilterDropdownMenuMobile } from './FilterDropdownMenu';
+import type { FilterButtonIconType, FilterDimension, FilterDropdownProps, FilterType } from './types';
+import { countActiveFilters, getFilterButtonClickHandler, resetAllFilters } from './utils';
+
+export const MainFilterDropdown = React.forwardRef<HTMLButtonElement, FilterDropdownProps>(
+  (
+    {
+      filterId = 0,
+      className,
+      filterButtonLabel,
+      buttonType,
+      handleClick,
+      filtersListState,
+      filters,
+      handleFilterSelection,
+      handleFilterUpdate,
+      clearFilterUpdate,
+      resultsCount,
+      ariaLabels = {},
+      id,
+    },
+    ref,
+  ) => {
+    const isButtonSelected = filtersListState?.[filterId] ?? false;
+    const { totalCount, filterCount } = countActiveFilters(filters, buttonType);
+
+    return (
+      <>
+        <FilterButtonDisplay
```

Comment:
I think you can change this to `<FilterButton/>` now since you've updated the Dropdown components